webpack/stories/docs/slot-and-fill.stories.mdx

```diff
@@ -13,9 +13,11 @@ Slot & Fill allows plugins to extend foreman core functionality in the UI
 
 ## Current Slots List
 
-| Name             |       Id        |                         Path |
-| ---------------- | :-------------: | ---------------------------: |
-| **About-footer** | aboutFooterSlot | _views/about/index.html.erb_ |
+| Name                 | Id               | Path  |
+| -------------------- |:----------------:| -----:|
+| **About-footer**     | aboutFooterSlot  | _views/about/index.html.erb_
+| **Host params**      | HostgroupParams  | _views/hosts/_form.html.erb_
+| **Hostgroup params** | HostParams       | _views/hostgroups/_form.html.erb_
```

Comment:
this is still swapped. First line - Name `Host params` first column and  Id `HostgroupParams` in second column.
```suggestion
| **Host params**      | HostParams  | _views/hosts/_form.html.erb_
| **Hostgroup params** | HostgroupParams       | _views/hostgroups/_form.html.erb_
```

Reply:
Should be fixed now.